config/default/kustomization.yaml

```diff
@@ -18,9 +18,9 @@ namePrefix: kudo-
 # YAML string, with resources separated by document
 # markers ("---").
 resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
+- rbac/rbac_role.yaml
```

Comment:
Reading through https://kubernetes.slack.com/archives/CG3HTFCMV/p1555327335097600 and https://github.com/kubernetes-sigs/kubebuilder/issues/654 should we use `./rbac/rbac_role.yaml` instead? I could just think of problems to arise about the location but right now both seem to look in the same location:

```
$ kustomize build config/default | kubectl apply -f -
Error: rawResources failed to read Resources: Load from path rbac/rbac_role.yaml failed: evalsymlink failure on '/Users/fabianbaier/go/src/github.com/fabianbaier/kudo/config/default/rbac/rbac_role.yaml' : lstat /Users/fabianbaier/go/src/github.com/fabianbaier/kudo/config/default/rbac: no such file or directory

error: no objects passed to apply
$ kustomize build config/default | kubectl apply -f -
Error: rawResources failed to read Resources: Load from path ./rbac/rbac_role.yaml failed: evalsymlink failure on '/Users/fabianbaier/go/src/github.com/fabianbaier/kudo/config/default/rbac/rbac_role.yaml' : lstat /Users/fabianbaier/go/src/github.com/fabianbaier/kudo/config/default/rbac: no such file or directory

error: no objects passed to apply
```